### PR TITLE
initial draft format guide

### PR DESCRIPTION
**Pull request by kmugrage**

### CODE REVIEW

File: content/format-guide.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+- There are two staff members required for each session, one facilator and one runner. The facilitator should be the same person who introduced the speaker.
+- When the talk is over, the facilator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
+- Each room will have 2 hand held microphones. The facilator should hold on to one of them, and the other should be handed to people who want to speak by the runner.
+- When the session is over, the facilator should do a short "thank you" wrap up so that the recording doesn't end ubruptly.
+
+Keep in mind:
+
+- The facilitator and runner should take great care to not let a couple people dominate the discussion.
+- Enfore the "don't talk if you're not holding a microphone" rule at all times.
```
Comment (juliegund) [inline content/format-guide.md]: Change to Enforce

File: content/format-guide.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- What challenges are you facing in this area? This question is designed to let people ask for help.
+- What successes have you had in this area? There are things people have done in every topic area that would benefit the others.
+- If you had a magic wand, what would you want to see in 2-5 years? There will be many people in attendance who can influence the tools and platforms available to us. Let them know what we need.
+
+Running the session:
+
+- There are two staff members required for each session, one facilator and one runner. The facilitator should be the same person who introduced the speaker.
+- When the talk is over, the facilator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
+- Each room will have 2 hand held microphones. The facilator should hold on to one of them, and the other should be handed to people who want to speak by the runner.
+- When the session is over, the facilator should do a short "thank you" wrap up so that the recording doesn't end ubruptly.
```
Comment (juliegund) [inline content/format-guide.md]: Change ubruptly to abruptly

File: content/format-guide.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+- What challenges are you facing in this area? This question is designed to let people ask for help.
+- What successes have you had in this area? There are things people have done in every topic area that would benefit the others.
+- If you had a magic wand, what would you want to see in 2-5 years? There will be many people in attendance who can influence the tools and platforms available to us. Let them know what we need.
+
+Running the session:
+
+- There are two staff members required for each session, one facilator and one runner. The facilitator should be the same person who introduced the speaker.
+- When the talk is over, the facilator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
+- Each room will have 2 hand held microphones. The facilator should hold on to one of them, and the other should be handed to people who want to speak by the runner.
```
Comment (juliegund) [inline content/format-guide.md]: Each room will have 2 hand held microphones. The facilitator should hold on to one of them, and the runner holds the other.

File: content/format-guide.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+- What challenges are you facing in this area? This question is designed to let people ask for help.
+- What successes have you had in this area? There are things people have done in every topic area that would benefit the others.
+- If you had a magic wand, what would you want to see in 2-5 years? There will be many people in attendance who can influence the tools and platforms available to us. Let them know what we need.
+
+Running the session:
+
+- There are two staff members required for each session, one facilitator and one runner. The facilitator should be the same person who introduced the speaker.
+- When the talk is over, the facilitator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
+- Each room will have 2 hand held microphones. The facilitator should hold on to one of them, and the runner holds the other.
```
Comment (kmugrage) [inline content/format-guide.md]: I still don't know how to resolve the two lines above. If we aren't asking people to leave their seat, the runner can't be holding the mic.

File: content/format-guide.md [hunk 5]
```diff
@@ -26,15 +26,15 @@ The questions:
 
 Running the session:
 
-- There are two staff members required for each session, one facilator and one runner. The facilitator should be the same person who introduced the speaker.
-- When the talk is over, the facilator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
-- Each room will have 2 hand held microphones. The facilator should hold on to one of them, and the runner holds the other.
+- There are two staff members required for each session, one facilitator and one runner. The facilitator should be the same person who introduced the speaker.
+- When the talk is over, the facilitator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
+- Each room will have 2 hand held microphones. The facilitator should hold on to one of them, and the runner holds the other.
 - When someone wants to speak, the runner holds the microphone up to their mouth so that they can. Do not hand the microphone over. Most people don't know how to hold one properly, and you might have to wrestle them to get it back.
```
Comment (markphahn) [inline content/format-guide.md]: Possibly, change the last three words to ". . . they can speak." e.g. When someone wants to speak, the runner holds the microphone up to their mouth so they can speak.

**Pull request merged**